tests/app/helpers/conftest.py

```diff
@@ -299,13 +300,103 @@ def expected_footer_nisra_theme():
                 ]
             }
         ],
-        "poweredBy": {
-            "logo": "nisra-logo",
-            "alt": "NISRA - Northern Ireland Statistics and Research Agency",
-        },
+        "poweredBy": (
```

Comment:
Is it possible to read the file here rather than having all of this set in the config?

Reply:
I have tried that but it failed the test because it didn't quite match what was returned you can see it and the unit test results in this commit [ac0d100](https://github.com/ONSdigital/eq-questionnaire-runner/pull/951/commits/ac0d100777b71c64f41ba4ea277f139dfe53c6a3)